app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardPresenter.java

```diff
@@ -134,7 +135,13 @@ public void toggleState() {
         event.state = Event.STATE_DRAFT.equals(event.state) ? Event.STATE_PUBLISHED : Event.STATE_DRAFT;
         eventRepository.updateEvent(event)
             .compose(dispose(getDisposable()))
-            .compose(progressiveErroneous(getView()))
+            .compose(ViewTransformers.progressive(getView()))
+            .doOnError(throwable -> {
+                if(ErrorUtils.getErrorDetails(throwable).getPointer().equals("starts-at"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['if' is not followed by whitespace.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=4963535)